cmd/zetae2e/local/local.go

```diff
@@ -370,6 +370,12 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	if !skipRegular {
 		startEVMTests(&eg, conf, deployerRunner, verbose)
 		startBitcoinTests(&eg, conf, deployerRunner, verbose, light, skipBitcoinSetup)
+		eg.Go(rpcTestRoutine(
```

Comment:
no need to execute these if its not in standard e2e test set, for example i notice them executed with solana tests so i moved it here